docs/conops.md

```diff
@@ -1,3 +1,178 @@
 # Concept of Operations - `svc-cargo`
 
-:construction:
+<center>
+
+<img src="https://github.com/Arrow-air/tf-github/raw/main/src/templates/doc-banner-services.png" style="height:250px" />
+
+</center>
+
+Item | Value
+--- | ---
+Maintainer(s) | @arrow-air/services
+Stuckee | A.M. Smith ([@ServiceDog](https://github.com/ServiceDog))
+
+:exclamation: This document is intended for internal use.
+
+## 1.0 Project Description
+
+#### 1.0.1 Background
+
+This service provides an interface for customers to schedule and manage cargo shipments. 
+
+It is not essential to the function of the core network. It exposes a public REST interface and translates valid requests from external clients into messages to the core network.
+
+#### 1.0.2 Overview of the Envisioned System
+
+<center> 
+
+```mermaid
+flowchart LR
+
+subgraph Client Domain
+    web[Website]
+    mobile[Mobile App]
+end
+
+subgraph Server
+    subgraph svc-cargo
+        rest{REST API}
+        logic((Logic))
+        grpc{GRPC Clients}
+
+        rest <--> logic
+        logic <--> grpc
+    end
+
+    pricing[svc-pricing]
+
+    subgraph Core Network
+        schedule[svc-scheduler]
+        storage[svc-storage]
+    end
+end
+
+web <---> rest
+mobile <---> rest
+grpc <-- Request/Confirm/Cancel Flight --> schedule
+grpc <-- Itinerary Costs --> pricing
+grpc <-- Get Vertiports --> storage
+```
+</center>
+
+`svc-cargo` exposes a REST API for external applications to make cargo-related requests. External applications include websites and mobile applications.
+
+Valid REST requests may trigger associated RPC calls to other microservices. Replies will route through `svc-cargo` and then to the external client. No external requests shall penetrate directly to the core network, nor shall replies from the core network route directly to the external client.
+
+A full description of interfaces can be found in the [ICD](./icd.md).
+
+## 2.0 Applicable Resources
+
+#### 2.0.1 Documents
+
+Document | Description
+--- | ---
+| [High-Level Concept of Operations (CONOPS)](https://github.com/Arrow-air/se-services/blob/develop/docs/conops.md) | Overview of Arrow microservices.                             |
+| [High-Level Interface Control Document (ICD)](https://github.com/Arrow-air/se-services/blob/develop/docs/icd.md)  | Interfaces and frameworks common to all Arrow microservices. |
+[Requirements - `svc-cargo`](https://docs.google.com/spreadsheets/d/1OliSp9BDvMuVvGmSRh1z_Z58QtjlSknLxGVdVZs2l7A/edit#gid=0) | Requirements and user stories for this microservice.
+[Software Design Document - `svc-cargo`](./sdd.md) | Defines the implementation of this interface.
+[Interface Control Document (ICD) - `svc-cargo`](./icd.md) | Defines the inputs and outputs of this microservice.
+
+## 3.0 Needs, Goals and Objectives of Envisioned System
+
+This service is expected to:
+- Handle many (100+) client requests per second for a region of operation.
```

Comment:
Have we tested this? I think the performance burden is more on the storage side to handle concurrent read/write ops.

Reply:
This is long term objectives

Reply:
I can remove the (100+) bit, figured "many" doesn't mean anything without context